rust/processor/src/db/common/models/default_models/raw_block_metadata_transactions.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::utils::util::{parse_timestamp, standardize_address};
+use aptos_protos::{transaction::v1::BlockMetadataTransaction, util::timestamp::Timestamp};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RawBlockMetadataTransaction {
+    pub version: i64,
+    pub block_height: i64,
+    pub id: String,
+    pub round: i64,
+    pub epoch: i64,
+    pub previous_block_votes_bitvec: String, // serde_json::Value,
+    pub proposer: String,
+    pub failed_proposer_indices: String, //serde_json::Value,
+    pub timestamp: chrono::NaiveDateTime,
+}
+
+impl RawBlockMetadataTransaction {
+    pub fn from_bmt_transaction(
```

Comment:
What was the point of refactoring this method to return the raw bmt? The method is only being used in the legacy postgres processor now

Reply:
isn't sdk version of default processor also processing bmt txns? and we need bmt for both parquet and postgres

Reply:
I wasn't able to find the usage of this implementation of `from_bmt_transaction` used elsewhere. But I see now that the sdk pg/pq processors import `default_processor::process_transactions`.

Reply:
yeah it's intentionally used in the one place within the `default_processor::process_transactions` so that both parquet and postgres can share the same implementation.